src/kv/LevelDBStore.cc

```diff
@@ -150,10 +150,10 @@ LevelDBStore::~LevelDBStore()
 {
   close();
   delete logger;
-  delete ceph_logger;
 
   // Ensure db is destroyed before dependent db_cache and filterpolicy
   db.reset();
+  delete ceph_logger;
```

Comment:
nit, we can also use `boost::scoped_ptr<>` to manage the life cycle of `ceph_logger`, just like how we use it to manage that of `db_cache` and `db`. but in that way, we need to move the ctor `LevelDBStore` to the .cc file.